recipes/sithom/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "sithom" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e256e0afa95184a548e346405248ab5abc948b35f490f239591787389b07d09c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - cmocean
+    - jupyterthemes
+    - matplotlib-base
+    - python >=3.8
+    - seaborn
+    - typeguard
+    - uncertainties
+    - xarray
+
+test:
+  imports:
+    - sithom
+  requires:
+    - pooch
+    - pytest
```

```suggestion
test:
  imports:
    - sithom
```
Aren't actually required because we aren't running those tests.